### PR TITLE
Add upgraded_org flow

### PR DESCRIPTION
**Pull request by jofsky**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

# Testing Notes
`cci flow run upgraded_org --org dev`

### CODE REVIEW

**Comment by jofsky** [comment]
@dgamboni

**Comment by dgamboni** [comment]
> @dgamboni would you mind reviewing this?

@jofsky Thanks for pinging me -- I'll add a Work Item so I can track the effort.

**lurch: add

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001Hs0cQAC" target="_blank">W-039293</a>

File: documentation/automation.md [hunk 1]
```diff
@@ -33,7 +33,7 @@
 | Regression                   | `regression_org`     | `release`        | ✔       |           |
 | Latest Trial Template        | N/A                  | `trial`          | ✔       |           |
 | Update Trialforce Source Org | `trial_org`          | `release`        | ✔       |           |
-
+| Upgraded Org                 | `upgraded_org`       | `dev`            |         |           |
```
Comment (jstvz) [inline documentation/automation.md]:
```suggestion
| Upgraded Org                 | `upgraded_org`       | `release`            |         |           |
```

**Comment by jofsky** [comment]
@jstvz

**Pull request merged**